### DISCUSSION TITLE
Guide for options

### DISCUSSION THREAD

**Discussion by JoshKasap** [Purfview/whisper-standalone-win#74] [Q&A] [answered]
Most of the advanced options descriptions aren't very obvious as to what they are and what they effect. 

Are you able to make a basic overview of what certain settings might do? 

Also are there any basic tips for options you might most commonly mess with when making srt. For example, how/when is initial prompt a good idea to use? If you plan on making a transcription from XYZ to English should the initial prompt be in English? Is this more of a prompt like chatgpt or more of and example text?

Also tips for example workflows. Like if transcribing an xyz video you may want to consider abc settings.

**Comment by Purfview** [comment, marked as answer]
> Most of the advanced options descriptions aren't very obvious as to what they are and what they effect.

The parameters are described same as they are described by the developers, search for more info at the original repos. [links are on the main page]

> Also are there any basic tips for options you might most commonly mess with when making srt.

Defaults are already good [in `whisper-faster.exe`], usually you don't need to mess with them, unless you have some problem or want it to run faster.

> For example, how/when is initial prompt a good idea to use?

IMO, when transcription is significantly inaccurate from a start. Then add the accurate words with prompt.

>  If you plan on making a transcription from XYZ to English should the initial prompt be in English?

I think it should be in original language, but I'm not sure.

> Is this more of a prompt like chatgpt or more of and example text?

An example text.
